website/docs/r/is_instance.html.markdown

```diff
@@ -286,6 +287,7 @@ Review the argument references that you can specify for your resource.
     
   - `image` conflicts with `boot_volume.0.snapshot`  
 - `keys` - (Optional, List) A comma-separated list of SSH keys that you want to add to your instance.
+- `metadata_service_enabled` - (Optional, Boolean) Indicates whether the metadata service endpoint is available to the virtual server instance. Default value : **false**
```

Comment:
Can you change the description ....Enables the metadata service endpoint on the virtual server instance if set to true(something like this instead of indicates)